Dockerfile

```
@@ -1,30 +1,36 @@
 FROM node:23.11.1-alpine3.21 AS base
 
+
+
 RUN apk add --no-cache \
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)